src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2464,16 +2464,16 @@ namespace NServiceBus.Sagas
     }
     public class SagaFinderDefinition
     {
-        public SagaFinderDefinition(System.Type type, string messageType, System.Collections.Generic.Dictionary<string, object> properties) { }
-        public string MessageType { get; }
+        public System.Type MessageType { get; }
+        public string MessageTypeName { get; }
```

Comment:
Yes, its value is MessageType.FullName.

Would you rather have it called `MessageTypeFullName`? I think an actual property is required to remove any ambiguity about what "name" is supposed to be used for comparisons to message headers, etc.